pkg/clients/redshift/redshift_test.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package redshift
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/redshift"
+	"github.com/google/go-cmp/cmp"
+
+	// runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+
+	"github.com/crossplane/provider-aws/apis/database/v1alpha1"
+	aws "github.com/crossplane/provider-aws/pkg/clients"
+)
+
+var (
+	dbName     = "example-name"
+	nodeType   = "dc1.large"
+	singleNode = "single-node"
+)
+
+func TestCreatePatch(t *testing.T) {
```

Comment:
Looks like a few functions are implicitly tested, but would like to see some explicit tests for something like `LateInitialize`

Reply:
Sure. Will add tests for those also.